.github/workflows/vm-kernel.yaml

```diff
@@ -28,24 +28,33 @@ on:
         type: boolean
         required: false
         default: false
+      archs:
+        description: 'Architectures to build for'
+        type: string
+        required: true
```

Comment:
question: The other workflows have a default here -- why not this one?

Reply:
added